controllers/discovery/searchNews.js

```diff
@@ -60,7 +64,10 @@ const Search = async (req, res) => {
           {
             model: DomainPage,
             as: 'newsLinkDomain',
-            attributes: ['domain_name', 'logo', 'credder_score', 'credder_last_checked']
+            attributes: ['domain_name', 'logo', 'credder_score', 'credder_last_checked'],
+            where: {
+              credder_score: {[Op.gte]: CREDDER_MIN_SCORE}
+            }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, you should add a check to ensure that `CREDDER_MIN_SCORE` is defined before using it in the query.

```diff
+            where: CREDDER_MIN_SCORE ? {
+              credder_score: {[Op.gte]: CREDDER_MIN_SCORE}
+            } : {}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
this is defined as constant, so will not be undefined